distances.ipynb

```diff
@@ -1,46 +1,73 @@
 {
```

Comment:
<p>as <code>pertpy.tl.Distance</code></p><p> </p><p>-&gt; in the module <code>pertpy.tl.Distance</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/pertpy-tutorials/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>